### ISSUE TITLE
Public count of tool and/or workflow runs

### ISSUE THREAD

**Issue by tshtatland** [galaxyproject/galaxy#18322]
It would be beneficial to developers of pipelines to know how many times a pipeline or tool that they developed is run by the members of the community.

The run count could be displayed for each version of the pipeline or as an aggregate count of all versions. The run count could be displayed on this page: https://usegalaxy.org/workflows/list_published and perhaps an additional sort button could be added, “Run count”.

**Comment by hexylena** [comment]
yep. it's why i wrote GRT a while back (which isn't used anymore) and some folks are working on other solutions that will be added.

partially covered by https://github.com/galaxyproject/galaxy/issues/13339 though I didn't consider a UI visible component which is a nice idea.

**Comment by mvdbeek** [comment]
I believe we can do this for tools, however this is a very hard task for workflows.

At this point we basically can't tell if a workflow is the same workflow (given that users can import workflows from various sources), let alone if a workflow is derived from another workflow. We're tracking that in https://github.com/galaxyproject/galaxy/issues/17536 and https://github.com/galaxyproject/galaxy/issues/11052. These are things we need to figure out first before it makes sense to expose these counts for workflows (and of course we can only do this for each individual galaxy instance).

**Comment by adrianreich** [comment]
I think that even just having a tool count would be valuable. Developers would love to be able to cite metrics of how the tools that they write are used across different platforms. There are even cases where highly specialized tools (or purpose built ones) where the count of the tool is able to serve as a proxy for a workflow count.
